<launch>
	<!-- HgROS -->
	<node name="HgROS" pkg="hg_cpp" type="hg_ros" output="screen">	
		<rosparam file="$(find hg_cpp)/params/test.yaml" command="load" />
		<param name="robot_description/ve026a" textfile="$(find ve026a_description)/robot/ve026a.urdf" />
		<param name="simulate" value="true"/>
  </node>
</launch>
